dbt_coves/tasks/generate/base.py

```diff
@@ -106,20 +109,27 @@ def get_metadata(self):
                                 self.get_metadata_map_key(row)
                             ] = self.get_metadata_map_item(row)
                         except KeyError as e:
-                            raise Exception(
-                                f"Key {e} not found in metadata file {path}"
-                            )
+                            raise Exception(f"Key {e} not found in metadata file {path}")
             except FileNotFoundError as e:
                 raise Exception(f"Metadata file not found: {e}")
 
         self.metadata = metadata_map
 
         return metadata_map
 
-    def render_templates(
-        self, relation, columns, destination, options=None, json_cols=None
-    ):
+    def get_config_value(self, key):
+        return self.coves_config.integrated["generate"][self.args.task][key]
+
+    def generate_template(self, destination_path, relation):
+        template_context = dict()
+        if "{{schema}}" in destination_path.replace(" ", ""):
```

Comment:
Replacing " " by "" in a path is a bug. In windows specially, it's very common to have folder names with spaces.

Reply:
You should use jinja.render